### PR TITLE
Update build paths to include src/bento/components

### PR DESCRIPTION
**Pull request by rileyajones**
As we migrate components within [amphtml](https://github.com/ampproject/amphtml) the location of their documentation is changing. This PR should ensure the new doc locations are picked up as well.

https://github.com/ampproject/amphtml/issues/36461

### CODE REVIEW

**Review by alanorozco — APPROVED** [review]
@sebastianbenz Could we get a review? 🙂

File: gulpfile.js/importDocs.js
Comment (alanorozco) [reply gulpfile.js/importDocs.js:151]: They'll all be moved, but there's some leftover. If it helps, we could move all of `README.md` and `example/` into `src/bento` before merging this.

Comment (sebastianbenz) [reply gulpfile.js/importDocs.js:151]: I was just curious. Works for me as is.

**Review by sebastianbenz — APPROVED** [review]
LGTM - thanks! Please let me know once all components have moved and we can remove the code for the extensions dir.

**Pull request merged**
